sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClient.java

```diff
@@ -398,8 +398,11 @@ public ConfigurationSetting setReadOnly(String key, String label, boolean isRead
 
     /**
      * Lists chronological/historical representation of {@link ConfigurationSetting} resource(s). Revisions are provided
-     * in descending order from their {@link ConfigurationSetting#getLastModified() lastModified} date. Revisions expire
-     * after a period of time. The service maintains change history for up to 7 days.
+     * in descending order from their {@link ConfigurationSetting#getLastModified() lastModified} date.
+     * Revisions expire after a period of time.
+     * <p>See <a href="https://azure.microsoft.com/en-us/pricing/details/app-configuration/">Pricing</a>
+     * for more information.</p>
```

Comment:
I think we should leave the last sentence as a single sentence, no need to break it up due to the hyperlink.

```suggestion
     * Revisions expire after a period of time, see <a href="https://azure.microsoft.com/en-us/pricing/details/app-configuration/">this documentation</a>
     * for more information.</p>
```